src/azure-cli/azure/cli/command_modules/storage/tests/latest/test_storage_azcopy_scenarios.py

```diff
@@ -11,6 +11,7 @@
 
 class StorageAzcopyTests(StorageScenarioMixin, LiveScenarioTest):
 
+    # test_resources_count = 0
```

Comment:
This is not needed as the fix is merged.
https://github.com/Azure/azure-cli/pull/10025/files